### PR TITLE
Fix save cover function to be dynamic

### PR DESCRIPTION
**Pull request by itsnameissteven**
#  Fix Saved Cover Issue
 ## Changes
* SaveCoverViewer innerHTML assigned to empty string
* For loop added to itterate each object in savedCovers array
* Fixes broken images stored on the saved covers page and allows for all saved covers to be viewed

### CODE REVIEW

**Review by ConnorAndersonLarson — CHANGES_REQUESTED** [review]
Can we remove the new event listener and wait to implement that in Iteration 5?

File: src/main.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ viewSavedCoversButton.addEventListener('click', changeToSavedCovers);
 homeButton.addEventListener('click', goHome)
 makeMyBook.addEventListener('click', makeBookCover)
 saveCoverButton.addEventListener('click', saveCoverData)
+savedCoverViewer.addEventListener('click', deleteCover)
```
Comment (ConnorAndersonLarson) [inline src/main.js]: This is the code in question to remove for now.

Comment (itsnameissteven) [reply src/main.js]: Will do!

Comment (ConnorAndersonLarson) [reply src/main.js]: Great! Resolved.

**Review by ConnorAndersonLarson — APPROVED** [review]

**Comment by ConnorAndersonLarson** [comment]
Cool, good learning experience today all around!

**Pull request merged**